src/components/exportAndImport/exportToJson.ts

```diff
@@ -13,42 +13,49 @@ const getRecordsAsObject = async ({
 
   const tablesToExport = filterByIncludeAndExclude(includeTables, excludeTables, databaseTables)
 
-  const fitleredRecords: ObjectRecords = {}
+  const filteredRecords: ObjectRecords = {}
   const records: ObjectRecords = {}
 
   for (const table of tablesToExport) {
     const { tableName } = table
-    const filteredReferences = getFilteredReferences(table, fitleredRecords)
-    const baseFilter = filters?.[tableName] || {}
+    const baseFilter = filters?.[tableName]
+    const filteredReferences = getFilteredReferences(table, filteredRecords, baseFilter)
 
     const { gql, getter, hasFilter } = constructGqlAndGetter(table, filteredReferences, baseFilter)
     const result = await databaseConnect.gqlQuery(gql)
     const rows = getter(result)
     records[tableName] = rows
-    if (hasFilter) fitleredRecords[tableName] = rows
+    if (hasFilter) filteredRecords[tableName] = rows
   }
 
   return records
 }
 
 const getFilteredReferences = (
   { referenceTables, columns }: DatabaseTable,
-  previousFilteredRecords: ObjectRecords
+  previousFilteredRecords: ObjectRecords,
+  baseFilter: object | undefined
 ) => {
   if (referenceTables.length === 0) return {}
 
-  // i.e. { sectionn: {id: {in: [10, 20]}}}
+  // i.e. { section: {id: {in: [10, 20]}}}
   const filters: {
     [referenceColumnName: string]: { in: any[] }
   } = {}
 
   columns.forEach(
     ({
       isReference,
+      isNullable,
       reference: { tableName: referencedTableName, columnName: referencedColumnName },
       columnName,
     }) => {
       if (!isReference) return
+      // If the column is nullable and there is already a base filter defined
+      // (in options) for this table, then don't make additional filters based
+      // on references, otherwise it can filter out to much (particularly for
+      // template export)
+      if (baseFilter && isNullable) return
```

Comment:
Added an additional check to only ignore filter references if the column is also nullable. 

The previous commit doesn't have this, it just ignores filter references if a baseFilter is present regardless of nullability: baae7b94 (See if you think this is better)